config/config.yaml

```diff
@@ -64,3 +64,8 @@ tunings:
     wait_duration: 60                                      # Duration to wait between each chaos scenario
     iterations: 1                                          # Number of times to execute the scenarios
     daemon_mode: False                                     # Iterations are set to infinity which means that the kraken will cause chaos forever
+telemetry:
```

Comment:
@tsebastiani let's add documents on the feature as well as the data being collected when enabled.